frontend/device/remarkable/device.lua

```diff
@@ -125,5 +129,220 @@ function Remarkable:reboot()
     os.execute("systemctl reboot")
 end
 
+local function osCapture(cmd, raw)
+    local f = io.popen(cmd, 'r')
+    if not f then
+        return nil
+    end
+    local s = f:read('*a')
+    f:close()
+    if not s then
+        return nil
+    end
+    if raw then
+        return s
+    end
+    s = string.gsub(s, '^%s+', '')
+    s = string.gsub(s, '%s+$', '')
+    s = string.gsub(s, '[\n\r]+', ' ')
+    return s
+end
+
+local function getNetworkProperty(path, name)
+    path = string.sub(path, string.len("/codes/eeems/oxide1/") + 1)
+    return rapidjson.decode(osCapture("rot --object Network:" .. path .. " wifi get " .. name))
+end
+local function getBSSProperty(path, name)
+    path = string.sub(path, string.len("/codes/eeems/oxide1/") + 1)
+    return rapidjson.decode(osCapture("rot --object BSS:" .. path .. " wifi get " .. name))
+end
+local function getWifiProperty(name)
+    return rapidjson.decode(osCapture("rot wifi get " .. name))
+end
+local function isempty(s)
+  return s == nil or s == ''
+end
+
+-- wireless
+function Remarkable:initNetworkManager(NetworkMgr)
+    if os.execute("which rot") == 1 then
+        logger.info("Could not find rot")
+        return
+    end
+    function NetworkMgr:turnOffWifi(complete_callback)
+        logger.info("Remarkable: disabling Wi-Fi")
+        os.execute("rot wifi call disable")
+        if complete_callback then
+            complete_callback()
+        end
+    end
+    function NetworkMgr:turnOnWifi(complete_callback)
+        logger.info("Remarkable: enabling Wi-Fi")
+        os.execute("rot wifi call enable")
+        local UIManager = require("ui/uimanager")
+        local InfoMessage = require("ui/widget/infomessage")
+        local info = InfoMessage:new{text = _("Turning on wifi…")}
```

Comment:
The message already elsewhere in the code is:
```suggestion
        local info = InfoMessage:new{text = _("Connecting to Wi-Fi…")}
```

Reply:
My bad, I was just skimming. In any case, unfortunately our current house style is Wi-Fi.

Reply:
Right, but I show another message `"Scanning for networks…"` after wifi properly enables. Should I just say `"Connecting to Wi-Fi..."` and not report the differences?

Reply:
I'm not sure if a message should really be shown only to be replaced by another in a split second, but turning on wifi is theoretically fine then.